packages/e2e.sandbox.todowidget/src/app.tsx

```diff
@@ -12,12 +12,13 @@ import {
   TextField,
 } from "@radix-ui/themes";
 import React, { useCallback, useEffect, useState } from "react";
-import { useFoundryContext } from "./context.js";
+import { useWidgetContext } from "./context.js";
 
 export const App: React.FC = () => {
-  const { parameterValues, hostEventTarget, emitEvent } = useFoundryContext();
+  const { parameterValues, parameterLoadingState, hostEventTarget, emitEvent } =
```

Comment:
i think `parameterValues` and `parameterLoadingState` is a bit long in the tooth still. maybe `parameters` and `parameters.state` and `parameters.values`